src/report/dot/theming.mjs

```diff
@@ -14,10 +13,14 @@ function matchesCriterion(pModuleKey, pCriterion) {
 
 function moduleOrDependencyMatchesCriteria(pSchemeEntry, pModule) {
   return Object.keys(pSchemeEntry.criteria).every((pKey) => {
-    // we use lodash.get here because in the criteria you can enter
-    // nested keys like "rules[0].severity" : "error", and lodash.get handles
+    // the keys can have paths in them like {"rules[0].severity": "info"}
+    // To get the criterion treat that key as a string and not as a path
+    // eslint-disable-next-line security/detect-object-injection
+    const lCriterion = pSchemeEntry.criteria[pKey];
+    // we use _.get here because in the criteria you can enter
+    // nested keys like "rules[0].severity" : "error", and _.get handles
     // that for us
```

Comment:
```suggestion
    // we use a bespoke 'get' here because in the criteria you can enter
    // nested keys like "rules[0].severity" : "error", and that function
    // handles those for us
```